motor/providers/local/local_unix.go

```diff
@@ -6,11 +6,19 @@ package local
 import (
 	"os"
 	"syscall"
+
+	pos "go.mondoo.com/cnquery/motor/providers/os"
 )
 
 func (p *Provider) fileowner(stat os.FileInfo) (int64, int64) {
 	uid := int64(-1)
 	gid := int64(-1)
+	if p.Sudo != nil {
```

Comment:
do we need to check for sudo? Are there other cases where we might have this `pos.FileInfo` type?

Reply:
You are correct, this also works without the sudo check. Removed it.